llvm/test/CodeGen/AArch64/GlobalISel/knownbits-const.mir

```diff
@@ -0,0 +1,29 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+# RUN: llc -mtriple aarch64 -passes="print<gisel-value-tracking>" %s -o - 2>&1 | FileCheck %s
+
+---
+name:            Cst
+tracksRegLiveness: true
+body:             |
+  bb.1:
+    ; CHECK-LABEL: name: Cst
+    ; CHECK-NEXT: KnownBits:00000001 SignBits:7 for %0:_
+    ; CHECK-NEXT: KnownBits:00000001 SignBits:7 for %1:_
+    %0:_(s8) = G_CONSTANT i8 1
+    %1:_(s8) = COPY %0
+...
+---
+name:            CstWithClass
+tracksRegLiveness: true
+body:             |
+  bb.1:
+    ; CHECK-LABEL: name: CstWithClass
+    ; We can't analyze %0 due to the register class constraint. We will get a
```

Comment:
The register class constraint is not a problem, the lack of an LLT is. Register class + LLT works fine